### PR TITLE
Update android_studio from 3.5.3.0 to 4.1.2.0

### PR DESCRIPTION
**Pull request by uberhacker**
Depends on #5347.

### CODE REVIEW

File: packages/android_studio.rb [hunk 1]
```diff
@@ -43,7 +43,7 @@ def self.postinstall
     puts 'settings, SDK and tools, execute the following:'.lightblue
     puts 'crew remove android_studio'.lightblue
     puts "rm -rf #{CREW_PREFIX}/.config/Android".lightblue
-    puts "rm -rf #{CREW_PREFIX}/.config/.AndroidStudio3.5".lightblue
```
Comment (supechicken) [inline packages/android_studio.rb:46]: Let `self.remove` do it?

**Review by supechicken — APPROVED** [review]

File: packages/android_studio.rb (hunk 1 above)
Comment (supechicken) [reply packages/android_studio.rb:46]: This line can be removed as the symlink installed in `CREW_DEST_HOME`

Comment (uberhacker) [reply packages/android_studio.rb:46]: Good call.

Comment (uberhacker) [reply packages/android_studio.rb:46]: Done.

**Comment by satmandu** [comment]
Are we sure that removing a config dir automatically is a good idea?

**Comment by uberhacker** [comment]
I can add a prompt.

**Pull request merged**